test/unit/server-configmap.bats

```diff
@@ -77,7 +82,8 @@ load _helpers
 
 @test "server/ConfigMap: namespace" {
   cd `chart_dir`
-  local actual=$(helm template \
+  local actual
+  actual=$(helm template \
```

Comment:
`local actual` is still being defined again on line 92.

Reply:
Fixed in 1697fde2b5e52fa53d120c4a7892b719494edcdf